packages/wdio-browserstack-service/src/accessibility-handler.ts

```diff
@@ -124,8 +141,12 @@ class _AccessibilityHandler {
             .filter((command) => command.name && command.class)
             .forEach((command) => {
                 const browser = this._browser as WebdriverIO.Browser
-                // @ts-expect-error fix type
-                browser.overwriteCommand(command.name, this.commandWrapper.bind(this, command), command.class === 'Element')
+                try {
+                    // @ts-expect-error fix type
+                    browser.overwriteCommand(command.name, this.commandWrapper.bind(this, command), command.class === 'Element')
+                } catch (error) {
+                    BStackLogger.debug(`Exception in overwrite command ${command.name} - ${error}`)
+                }
```

Comment:
No need for try/catch here
```suggestion
                browser.overwriteCommand(command.name, this.commandWrapper.bind(this, command), command.class === 'Element')
```

Reply:
This is not resolved, there should be no try/catch around the `overwriteCommand`

Reply:
So these overwrite commands will be sent from the server. And in future, if some command is removed that shouldn't cause the user test script to fail that's why we have added this try catch for overwrite command

Reply:
I don't think this is a good idea to handle this but I will leave that decision up to your team.

Reply:
Is there anything that pending on this PR? If not when can we merge this